hack/update/kubernetes_version/update_kubernetes_version.go

```diff
@@ -101,6 +101,11 @@ var (
 				`kubernetesVersion:.*`: `kubernetesVersion: {{.LatestVersionP0}}`,
 			},
 		},
+		"deploy/kicbase/Dockerfile": {
+			Replace: map[string]string{
+				`ARG KUBERNETES_VERSION=.*`: `ARG KUBERNETES_VERSION="{{.StableVersion}}"`,
```

Comment:
The kicbase _cannot_ have a single kubernetes version, if it is going to be reused for multiple versions.

Reply:
Would it work for more cases if this kubernetes sources.list is then replaced by later tooling, when the specific version of k8s is installed?

Reply:
The easiest would probably be to install the CRI adapter with the Kubernetes components (like cri-tools/cni-plugins/kubeadm/kubelet/kubectl) - possibly split it out into per container runtime, like the "preload" is now?

Reply:
It seems like CRI-O now uses an internal bundling of crun and conmon, so it doesn't use the system installation of those anymore,,, Otherwise such shared things could go with the OS installation still, if not k8s-version specific.

Reply:
Heh, the yak shaving continues (the path that led me here is quite convoluted already!), I'll check out the preload alternative.

Reply:
Added CRI-O to the discuss items for today's meeting, since there was already WIP for it

Reply:
Experimented with building static podman with Nix, similar to the cri-o packaging... ([discussion](https://github.com/containers/podman/discussions/23743))